toolkit/tools/internal/testutils/testrpms/downloader/Dockerfile.azurelinux

```diff
@@ -0,0 +1,18 @@
+ARG baseimage
+FROM ${baseimage}
+ARG imagecreator
```

Comment:
Remove the `imagecreator` ARG and handle `jq golang` in `download-test-rpms.sh`. That way, we simplify this file.